src/trace-api.js

```diff
@@ -177,6 +178,22 @@ TraceAgent.prototype.runInRootSpan = function(options, fn) {
 };
 
 /**
+ * Returns a unique identifier for the current root span. This can be used to
+ * uniquely identify the current root span. If there is no current root span,
+ * or if we have lost context, this will return null. The structure and the
+ * length of the returned string should be treated opaquely - the only guarantee
+ * is that the value would unique for every root span.
+ * @returns {string} the root span id, or null if it doesn't exist
+ */
+TraceAgent.prototype.getRootSpanId = function() {
```

Comment:
I like `getCurrentContextId` as it does feel pretty general and in the situation where we propagate context without trace sampling being enabled, it helps subtly clarify that there is not actually a trace associated with this ID even though we are passing around the context.